tools/flaky-test-retryer/README.md

```diff
@@ -97,6 +97,14 @@ automatic retry.
 
 if all retries are expended without a success.
 
+## Updating
+
+1. Run `make push_versioned` in `/images/flaky-test-retryer/` with a clean working tree.
+2. Note the image tag, in the format `v[date]-[hash]`.
+3. Replace the current image tag in [retryer_service.yaml](https://github.com/knative/test-infra/blob/master/tools/flaky-test-retryer/gke_deployment/retryer_service.yaml#L40)
```

Comment:
Use a permalink, so the link won't point to the wrong entry if this file changes.

```suggestion
3. Replace the current image tag in [retryer_service.yaml](https://github.com/knative/test-infra/blob/751583ee73a463cf6449817afda38d63b29dfff7/tools/flaky-test-retryer/gke_deployment/retryer_service.yaml#L40)
```